main.go

```diff
@@ -220,7 +220,22 @@ func checkForGitAndDockerHub(allFiles []parser_struct.UpdatedRepo, spec *godo.Ap
 	}
 }
 
-// filterApps filters git and DockerHub apps and then updates app spec with DOCR
+func makeImageSpec(updatedRepo parser_struct.UpdatedRepo) *godo.ImageSourceSpec {
+
+	if updatedRepo.Image.RegistryType == "" {
```

Comment:
Do we need to check `Image != nil` for legacy format. Or if the getter is nil safe just do `updatedRepo.GetImage().RegistryType`

Reply:
good callout, will fix

Reply:
actually, There is no `GetImage()` for that type. `updatedRepo.Image` gets initialized to the zero value, so I don't see another way to check other than what I have.

<img width="999" alt="image" src="https://user-images.githubusercontent.com/234704/157301336-e8be39a8-0e12-4644-b1ec-3b73846e0603.png">

Reply:
Yep -- that makes sense; I had mistaken it for a pointer